client/test/explorer/gatewayExplorer.test.ts

```diff
@@ -445,11 +449,23 @@ ${FabricWalletUtil.LOCAL_WALLET}`);
             it('should create a connected tree if there is a connection', async () => {
                 allChildren.length.should.equal(3);
 
-                const connectedItem: ConnectedTreeItem = allChildren[0] as ConnectedTreeItem;
-                connectedItem.label.should.equal('Connected via gateway: myGateway');
-                connectedItem.contextValue.should.equal('blockchain-connected-item');
-                connectedItem.collapsibleState.should.equal(vscode.TreeItemCollapsibleState.None);
-                connectedItem.connection.name.should.equal('myGateway');
+                const connectedItem1: ConnectedTreeItem = allChildren[0] as ConnectedTreeItem;
+                connectedItem1.label.should.equal('Connected via gateway: myGateway');
+                connectedItem1.contextValue.should.equal('blockchain-connected-item');
+                connectedItem1.collapsibleState.should.equal(vscode.TreeItemCollapsibleState.None);
+                connectedItem1.connection.name.should.equal('myGateway');
+
+                const connectedItem2: ConnectedTreeItem = allChildren[1] as ConnectedTreeItem;
+                connectedItem2.label.should.equal('Using ID: undefined');
```

Comment:
can you set the `fabricConnection.identityName` to a random name in this describe, and then check that it's in the label here? as opposed to checking for undefined.